writeToStream.js

```diff
@@ -41,6 +41,7 @@ function generate (packet, stream) {
       return false
   }
 }
+generate.cacheNumbers = true
```

Comment:
Then it would require:
either load and generate cache and then potentially clean it
or set `cacheNumbers` always (either true or false) to trigger the selection of required `writeNumber`.